L1Trigger/L1THGCal/python/hgcalConcentratorProducer_cfi.py

```diff
@@ -29,14 +31,42 @@
                                0,  0,  0,  0,   0,   0, 0, 0,
                                0,  0,  0,  0,   0,   0, 0, 0,
                               ),
+                          coarsenTriggerCells = cms.bool(False),
+                          fixedDataSizePerHGCROC = cms.bool(False),
                           )
 
 
 supertc_conc_proc = cms.PSet(ProcessorName  = cms.string('HGCalConcentratorProcessorSelection'),
                              Method = cms.string('superTriggerCellSelect'),
-                             stcSize = cms.vuint32(4,4,4)
+                             type_energy_division = cms.string('superTriggerCell'),# superTriggerCell,oneBitFraction,equalShare
+                             stcSize = cms.vuint32(4,4,4,4),
```

Comment:
it was intended (since the 4,8,8,8 is typically used with `fixedDataSizePerHGCROC=True`, whilst the `superTriggerCell` has traditionally been used with a size of 4 everywhere. However I can change to be consistent if needed.

Reply:
We should make sure that the values used for performance studies are consistent with the number of links that will quoted at the June workshop. 
So all the parameter values should be realistic in terms of corresponding number of links.

Reply:
Ok - I will be presenting results with 4,16,16,16 (`fixedDataSizePerHGCROC=False`) and 4,8,8,8 (`fixedDataSizePerHGCROC=True`). So I think it should be ok.

Reply:
`4,4,4,4` is neither `4,8,8,8` nor `4,16,16,16`. If `4,4,4,4` does not correspond to any realistic scenario, it shouln't be the default any more.